### DISCUSSION TITLE
Why is the prompt information of the chart deleted in the UI of 9.3.0?

### DISCUSSION THREAD

**Discussion by saber-wang** [apache/skywalking#10151] [Q&A] [answered]
9.1.0:
![image](https://user-images.githubusercontent.com/45062099/207242150-e7876972-350a-4489-925d-e32ffeb6bf5f.png)

9.3.0
![image](https://user-images.githubusercontent.com/45062099/207242227-00466bb0-3243-4232-91e8-c65a66ba8251.png)

In 9.3.0, you must place the cursor on the vertex to estimate the current value

**Comment by wu-sheng** [comment]
Due to support metrics association, I think. Those two styles are conflicts in charts, AFAIK. @Fine0830 right?

**Comment by Fine0830** [comment, marked as answer]
Right. I will optimize this interaction to make it more friendly.

**Reply by wu-sheng** [reply]
https://github.com/apache/skywalking/issues/10206
